### PR TITLE
Adding composer.json

### PR DESCRIPTION
**Pull request by guillemmont**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "restgames/battleship-rest-silex-skeleton",
+    "description": "Battleship REST Service Skeleton using Silex and PHP",
+    "homepage": "https://github.com/restgames/battleship-rest-silex-skeleton",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Carlos Buenosvinos",
```
Comment (jcuerdo) [inline composer.json]: Please put your name :)

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            "email": "carlos.buenosvinos@gmail.com",
```
Comment (jcuerdo) [inline composer.json]: Email also

File: composer.json (hunk 1 above)
Comment (guillemmont) [inline composer.json]: ya veremos

File: composer.json (hunk 2 above)
Comment (guillemmont) [inline composer.json]: ok

**Pull request merged**